spec/system/index_spec.rb

```diff
@@ -14,10 +14,12 @@
 
     it "includes the shows titles" do 
       visit "/" 
-      csv_data.first(5).each do |show|
-        expect(page).to have_text(show['title'])
+      Show.all.first(5).each do |show|
```

Comment:
you should use fixtures here, like factory bot or so. You should not seed the database with the real csv file here. Btw how are you even seeding it :o

Reply:
This PR was quite outdated sorry about that. I should have left it as draft PR instead ready for review. 
Now is almost up to date! 👍 Thank you for the comments Goran!